app.js

```diff
@@ -0,0 +1,38 @@
+const express = require("express");
+const app = express();
+const userRouter = require("./route/userRoute");
+const bodyParser = require('body-parser');
+const mongoose = require("mongoose");
+
+app.use(bodyParser.json());
+
+app.use((req, res, next) => {
+    res.setHeader('Access-Control-Allow-Origin', '*');
+    res.setHeader(
+        'Access-Control-Allow-Methods',
+        'OPTIONS, GET, POST, PUT, PATCH, DELETE'
+    );
+    res.setHeader('Access-Control-Allow-Headers', 'Content-Type, Authorization');
+    if (req.method === 'OPTIONS') {
+        return res.sendStatus(200);
+    }
+    next();
+});
+
+app.use(userRouter);
+
+app.use((error, req, res, next) => {
+    const status = error.statusCode || 500;
+    const message = error.message;
+    const data = error.data;
+    res.status(status).json({ message: message, data: data });
+});
+
+mongoose
+    .connect(
+        'mongodb+srv://root:root@testcluster.w3klz.mongodb.net/jwtDb?retryWrites=true&w=majority'
```

Comment:
This should not be revealed in your source code. Instead you need to read this from `.env` file as follows:

// in .env file you have:
MONGO_URL=mongodb+srv://root:root@testcluster.w3klz.mongodb.net/jwtDb?retryWrites=true&w=majority

// In app.js you have:
require('dotenv').config();
...

mongoose.connect(process.env.MONGO_URL);